.github/actions/generate-resources/main.js

```diff
@@ -78,19 +78,11 @@ const init = async () => {
 
   // Write README.md file
   fs.writeFileSync(
-    './README.md',
+    './RESOURCES.md',
```

Comment:
I saw that it was originally `RESOURCES.md` before, I also made this change temporarily just to verify that it's generating the new file.